backends/advanced/src/advanced_omi_backend/services/obsidian_service.py

```
@@ -0,0 +1,386 @@
+            FileNotFoundError: If vault path doesn't exist.
+        """
+        if not os.path.exists(vault_path):
+            raise FileNotFoundError(f"Vault path not found: {vault_path}")
+
+        self.setup_database()
+        
+        processed = 0
+        errors = []
+        
+        for root, dirs, files in os.walk(vault_path):
+            dirs[:] = [d for d in dirs if not d.startswith('.')] # noqa
```

Ruff (0.14.8)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)